eth2near/eth2near-block-relay-rs/src/eth2near_relay.rs

```diff
@@ -285,6 +292,13 @@ impl Eth2NearRelay {
                     .header_update
                     .beacon_header
                     .slot;
+
+                if finality_update_slot <= last_finalized_slot_on_near {
+                    info!(target: "relay", "Finality update slot for hand made light client update <= last finality update on near. Increment gap for signature slot and skipping light client update.");
+                    self.current_gap_between_finalized_and_signature_slot += 32;
```

Comment:
Use a constant please

Reply:
fixed: https://github.com/aurora-is-near/rainbow-bridge/pull/775/commits/84e365d06b00ec476f34b107149d26b610b2a4af